### ISSUE TITLE
Lots of hypothetical protein annotations for Plasmid Sequences

### ISSUE THREAD

**Issue by newcanu** [tseemann/prokka#351]
I am trying to annotate a series of plasmids from K. pneumoniae and E. coli isolates using Prokka. However a large amount of the annotations given are "hypothetical proteins". Are there any suggestions to improve annotation for plasmid sequences? I have tried downloading 3-4 similar plasmids from genbank and using the --protein command to link to them. However 52/88 annotations are still listed as hypothetical proteins.

**Comment by tseemann** [comment]
You did the correct thing by downloading relevant plasmids and using `--proteins`

Feel free to use all the plasmids in Genbank even - just download them and concatenate the .gbk files.

There are still *many* unknown proteins in the pan bacterial proteome so I am not surprised there is 1/3 hypothetical.

**Comment by graceblackwell** [comment]
I think I am having the same problem. I performed a test where I ran prokka using the genbank annotations of AF550415 against it's fasta sequence and many of the named genes from the genbank file were now called hypothetical proteins, particularly in the transfer operon (tra regions). Command was `prokka --protein AFF50415.gb  --outdir self_test --prefix out AFF50415.fasta`

log file
[out.log](https://github.com/tseemann/prokka/files/4213515/out.log)
annotation tsv summary
[out_2.txt](https://github.com/tseemann/prokka/files/4213524/out_2.txt)

Any help would be appreciated.
